src/content/reference/react/hooks.md

```diff
@@ -106,24 +106,6 @@ To prioritize rendering, use one of these Hooks:
 
 ---
 
-## Resource Hooks {/*resource-hooks*/}
```

Comment:
Can we move this to apis.md? It can also explain that it can only be called in render, like a hook, but can be called conditionally, unlike a hook.